Vimari Extension/js/settings.js

```diff
@@ -17,6 +17,7 @@ function getSettings() {
         'scrollDownHalfPage': 'd',
         'goToPageTop': 'g g',
         'goToPageBottom': 'shift+g',
+        'goToFirstInput': 'g i',
```

Comment:
This line is causing a conflict as `js/settings.js` is replaced by `json/defaultSettings.json`. If you would be able to add this line there instead I would like to merge this.